src/style.css

```diff
@@ -13,10 +9,12 @@
 .container_1 {
   width: 50%;
   height: 50%;
-  border: 5px solid #c3c3c3;
+  border: 5px solid #af3636;
+  border-radius: 30px;
   display: flex;
   flex-direction: column;
   justify-content: center;
+  background-color: burlywood;
 }
```

Comment:
- [ ] Hi @jugosack, great effort working on the styling of your leaderboard, currently, I noticed some differences in your design compared with the wireframe given to us. The `submit` button is not well aligned with the form and it is moving out of the container, Kindly fix this and you'll be sure to have a perfect design.

Your design 👇 

![yugo](https://user-images.githubusercontent.com/78634841/216692457-3b3f96d6-1f2e-4564-a998-792912596f63.png)

Wireframe 👇 

<img src="https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png?raw=true">